lib/aptible/cli/subcommands/logs.rb

```diff
@@ -25,6 +28,149 @@ def logs
               ENV['ACCESS_TOKEN'] = fetch_token
               exit_with_ssh_portal(op, '-o', 'SendEnv=ACCESS_TOKEN', '-T')
             end
+
+            desc 'logs_from_archive --bucket NAME | --aws-region REGION | ' \
```

Comment:
The option is `--region`.

```suggestion
            desc 'logs_from_archive --bucket NAME | --region REGION | ' \
```